internal/worktreeprep/fixture_test.go

```
@@ -0,0 +1,85 @@
+}
+
+func (fixture repositoryFixture) createRemoteBranch(t *testing.T, branch string) {
+	t.Helper()
+	fixture.createLocalBranch(t, branch)
+	gitCommand(t, fixture.primary, "push", "origin", branch)
+	gitCommand(t, fixture.primary, "branch", "-d", branch)
+}
+
+func gitCommand(t *testing.T, cwd string, args ...string) string {
+	t.Helper()
+	command := exec.Command("git", args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)